### ISSUE TITLE
Form elements do not have associated labels

### ISSUE THREAD

**Issue by Aircl0wn** [GoogleChrome/lighthouse#8839]
#### Provide the steps to reproduce
1. Run LH on https://www.veenendaal.nl/

```
<form action="https://www.veenendaal.nl/nc/algemeen/zoeken/" method="get" class="search-bar-form">
    <label for="searchBar" class="sr-only">Doorzoek de site</label>
    <input type="search" id="searchBar" name="q" class="search-bar-input" value="" placeholder="Zoeken" title="invoerveld zoekopdracht">
    <button type="submit" class="search-bar-button" id="search-bar-button" aria-label="Start met zoeken">
        <span class="search-button-text">Zoeken</span>
    </button>
</form>
```

#### What is the current behavior?
Lighthouse (in Chrome and separate extension) returns notification of a missing label for the input field. Axe does not return this notification.

This does not happen if i test the snippet above separately so it's a combination of content on the page that's causing lighthouse to fail. So far i haven't figured out why specifically.

#### What is the expected behavior?
The input field has a label provided with the for attribute set to the correct id of the input field.

#### Environment Information
* Affected Channels: Extension, DevTools tested
* Lighthouse version: 4.3.1
* Node.js version: -
* Operating System: MacOS 10.14.4, Chrome 74.0.3729.131

**Related issues**

**Comment by patrickhulce** [comment]
Thanks for filing! I'm not seeing this on our 5.0 prerelease where we've updated axe, so my guess would be it was a bug in the older version of axe.

![image](https://user-images.githubusercontent.com/2301202/57162192-7b82ea80-6dbb-11e9-803e-ebb056184e37.png)

**Comment by brendankenny** [comment]
looks like the fix was in https://github.com/dequelabs/axe-core/pull/1187, released in aXe 3.2.0, landed in Lighthouse in #8370, and, yah, will be in 5.0

**Comment by brendankenny** [comment]
> looks like the fix was in dequelabs/axe-core#1187,

well, looks like that's actually for the issue in #8435, but the rest of the comment is correct :)

**Comment by Aircl0wn** [comment]
wonderful, thanks guys!

**Comment by connorjclark** [comment]
This should have landed in 5.0
